packages/jest-runtime/src/transform.js

```diff
@@ -19,10 +19,12 @@ const getCacheFilePath = require('jest-haste-map').getCacheFilePath;
 const path = require('path');
 const stableStringify = require('json-stable-stringify');
 const vm = require('vm');
+const {shouldBeCovered} = require('jest-config');
 
 const VERSION = require('../package.json').version;
 
 export type Processor = {
```

Comment:
weird, this type is missing the `getCacheKey` function and has a wrong signature for `process`.